cfme/configure/settings.py

```diff
@@ -1,424 +1,312 @@
+    container_templates = ViewButtonGroup('Containers', 'Templates')
+    vm_visibility = BootstrapSwitch('display_vms')
+    save = Button('Save')
 
-    def copy(self, name=None):
-        view = navigate_to(self, 'All')
-        rows = view.timeprofile_form.table
-        for row in rows:
-            if row.description.text == self.description:
-                row[0].check()
-        view.timeprofile_form.configuration.item_select('Copy selected Time Profile')
-        if name is not None:
-            new_timeprofile = Timeprofile(description=name, scope=self.scope)
-            changed = view.timeprofile_form.fill({
-                'description': name,
-                'scope': self.scope,
-            })
-        else:
-            new_timeprofile = Timeprofile(description="{} copy".format(self.description),
-                                          scope=self.scope)
-            changed = view.timeprofile_form.fill({
-                'description': "{} copy".format(self.description),
-                'scope': self.scope,
-            })
 
-        if changed:
-            view.timeprofile_form.save_button.click()
-        return new_timeprofile
-
-    def delete(self):
-        view = navigate_to(self, 'All')
-        rows = view.timeprofile_form.table
-        for row in rows:
-            if row.description.text == self.description:
-                row[0].check()
-        view.timeprofile_form.configuration.item_select("Delete selected "
-                                                        "Time Profiles", handle_alert=True)
+class DefaultFiltersForm(View):
+    tree = CheckableBootstrapTreeview('df_treebox')
+    save = Button('Save')
+
 
+class TimeProfilesView(View):
+    table = Table("//div[@id='main_div']//table")
+    flash = FlashMessages('.//div[@id="flash_msg_div"]'
+                          '/div[@id="flash_text_div" or contains(@class, "flash_text_div")]')
 
-@navigator.register(Timeprofile, 'All')
-class TimeprofileAll(CFMENavigateStep):
-    VIEW = TimeProfileAddFormView
-    prerequisite = NavigateToAttribute('appliance.server', 'MySettings')
 
-    def step(self):
-        self.view.mysetting.tabs.time_profile.select()
+class TimeProfileAddView(BaseLoggedInPage):
+    title = Text('//div[@id="main-content"]//h3')
+
+    description = Input(id='description')
+    scope = BootstrapSelect('profile_type')
+    timezone = BootstrapSelect('profile_tz')
+    days = BootstrapSwitch(name='all_days')
+    hours = BootstrapSwitch(name='all_hours')
+    save_button = Button(VersionPick({
+        Version.lowest(): 'Add',
+        '5.8': 'Save',
+        '5.9': 'Add'
+    }))
+    save_edit_button = Button('Save')
+    cancel_button = Button('Cancel')
+    help_block = Text("//span[contains(@class, 'help-block')]")
+
+    @property
+    def is_displayed(self):
+        """Is this page displayed?"""
+        return self.description.is_displayed
 
 
 class Visual(Updateable, Navigatable):
```

Comment:
**Required**: Navigatable is deprecated, please use NavigatableMixin

Reply:
Thanks. I saw both being used and wasn't sure which was correct, so I just left this as-is.